mesh/profiles/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from . import views as profile_views
+from django.http import HttpRequest
+
+urlpatterns = [
+    
+    # Home Path for Example API 'exampleapi'
+    path('biography/', profile_views.biography, name="biography"),
```

Comment:
What is this biography stuff? How is it related to tags table?

Reply:
Sorry about that, I do not remember replacing it with that... will fix